webapp/src/blocks.tsx

```diff
@@ -385,14 +385,10 @@ export class Editor extends toolboxeditor.ToolboxEditor {
         });
 
         for (const block in deprecatedMap) {
-            if (deprecatedMap[block] === 0) {
-                delete deprecatedMap[block];
+            if (deprecatedMap[block] !== 0) {
+                pxt.tickEvent("blocks.usingDeprecated", {block : block, count : deprecatedMap[block] });
```

Comment:
typically lowercased?

Reply:
I didn't change the tick name. Kept is as before. I can update next round